_maps/map_files/cerestation/cerestation.dmm

```diff
@@ -5678,14 +5634,6 @@
 	icon_state = "dark"
 	},
 /area/station/security/evidence)
-"aJE" = (
-/obj/effect/turf_decal/stripes/line{
-	dir = 8
-	},
-/turf/simulated/floor/plasteel{
-	dir = 8
-	},
-/area/station/supply/miningdock)
```

Comment:
This is an example of it removing redundant ones?

Reply:
Correct; without the dir=8 on the floor tile, this becomes `/obj/effect/turf_decal/stripes/line{dir=8},/turf/simulated/floor/plasteel,/area/station/supply/miningdock`, which exists in the prefab at line L48119 already, so we can toss this whole prefab.